charts/hedera-mirror-importer/values.yaml

```
@@ -468,6 +468,15 @@ serviceAccount:
   # The name of the service account to use. If not set and create is true, a name is generated using the fullname template
   name:
 
+startupProbe:
+  failureThreshold: 360
+  httpGet:
+    path: /actuator/health/readiness
```

We should create a separate actuator endpoint just for startup that only includes migration. Migration should be removed from readiness.

```suggestion
    path: /actuator/health/startup
```